python/hail/linalg/blockmatrix.py

```diff
@@ -119,14 +123,12 @@ def fromfile(cls, uri, n_rows, n_cols, block_size=None):
 
         Examples
         --------
-
         >>> import numpy as np
         >>> a = np.random.rand(10, 20)
         >>> a.tofile('/local/file') # doctest: +SKIP
```

Comment:
These are skipped because I want to emphasize that a.tofile is just a numpy command writing to a local file. Everywhere else '/output/file' is a Hadoop path. But they are tested in `test_to_from_numpy`.